carbonmark-api/src/utils/helpers/utils.ts

```diff
@@ -275,3 +275,17 @@ export function asResponse<ReplyType>(
     .header("Content-Type", "application/json; charset=utf-8")
     .send(payload);
 }
+
+/**
+ * Creates a GQL query fragment from an object
+ * @param obj_from_json
+ * @returns
+ */
+export function gqlWhereFragment<T>(obj: T) {
+  // See https://stackoverflow.com/questions/11233498/json-stringify-without-quotes-on-properties
+  // JSON stringify
+  let json = JSON.stringify(obj);
+  // Remove double quotes around attribute names
+  json.replace(/\\"/g, "\uFFFF"); // U+ FFFF
+  return (json = json.replace(/"([^"]+)":/g, "$1:").replace(/\uFFFF/g, '\\"'));
+}
```

Comment:
I can't see this being used anywhere?

Reply:
It is in [carbonmark-api/src/routes/retirements/get.ts](https://github.com/KlimaDAO/klimadao/pull/2034/files/5ea83c65f65f4b4794be22d5749045e68a1812a6#diff-fc9cd5d99f9a8cc740683d1aa0e159094aa7c770d59f9f7c6f77daa41596c94e)